src/util/native_fs_utils.js

```diff
@@ -432,15 +432,18 @@ function isDirectory(ent) {
 }
 
 /**
- * @param {string} [config_root_backend]
+ * @param {string} [backend]
+ * @param {number} [warn_threshold_ms]
+ * @param {Function} [report_fs_stats]
  * @returns {nb.NativeFSContext}
  */
-function get_process_fs_context(config_root_backend) {
+function get_process_fs_context(backend, warn_threshold_ms, report_fs_stats) {
     return {
         uid: process.getuid(),
         gid: process.getgid(),
-        warn_threshold_ms: config.NSFS_WARN_THRESHOLD_MS,
-        backend: config_root_backend
+        warn_threshold_ms: warn_threshold_ms || config.NSFS_WARN_THRESHOLD_MS,
+        backend: backend,
+        report_fs_stats
```

Comment:
Does the name of the function is always `report_fs_stats`?
Did understand why it is not:
```suggestion
        report_fs_stats: report_fs_stats
```

Reply:
not sure I understood the question... what's the difference between the current line and the suggested one?
```
> const x = 1;
> console.log({x : x});
{ x: 1 }
undefined
> console.log({x});
{ x: 1 }
```

Reply:
I don't remember what I meant, sorry, resolving.